app/helpers/dsfr_link_helper.rb

```diff
@@ -1,113 +1,51 @@
 module DsfrLinkHelper
-  LINK_STYLES = {
-    inverse:          "fr-link--inverse",
-    muted:            "fr-link--muted",
-    no_underline:     "fr-link--no-underline",
-    no_visited_state: "fr-link--no-visited-state",
-    text_colour:      "fr-link--text-colour",
-  }.freeze
-
-  BUTTON_STYLES = {
-    disabled:  "fr-btn--disabled",
-    secondary: "fr-btn--secondary",
-    warning:   "fr-btn--warning",
-  }.freeze
-
-  def dsfr_link_classes(*styles, default_class: 'fr-link')
-    if (invalid_styles = (styles - LINK_STYLES.keys)) && invalid_styles.any?
-      fail(ArgumentError, "invalid styles #{invalid_styles.to_sentence}. Valid styles are #{LINK_STYLES.keys.to_sentence}")
-    end
-
-    [default_class] + LINK_STYLES.values_at(*styles).compact
-  end
-
-  def dsfr_button_classes(*styles, default_class: 'fr-btn')
-    if (invalid_styles = (styles - BUTTON_STYLES.keys)) && invalid_styles.any?
-      fail(ArgumentError, "invalid styles #{invalid_styles.to_sentence}. Valid styles are #{BUTTON_STYLES.keys.to_sentence}")
-    end
-
-    [default_class] + BUTTON_STYLES.values_at(*styles).compact
-  end
+  ICON_SIDES = %w(left right).freeze
```

Comment:
I'm guessing `position` would be a better name, wdyt?

Reply:
agreed